docs/cli/server.md

```diff
@@ -22,7 +22,7 @@ The following options must be set to the same value on all servers in the cluste
 * `--disable-cloud-controller`
 * `--disable-helm-controller`
 * `--disable-network-policy`
-* `--disable-servicelb`
+* `--disable servicelb`
```

Comment:
nit: Just delete the flag from here; the `--disable` flag is covered elsewhere.
```suggestion
```

Reply:
All of the flags in this section ([Critical Configuration Values][1]) are covered elsewhere, but it appears they are listed here because they "must be set to the same value on all servers in the cluster." I'm inclined to leave it in this list if it truly must be the same on all nodes (I don't know whether or not that it true for `--disable servicelb`), like the other flags listed here.

[1]: https://docs.k3s.io/cli/server#critical-configuration-values

Reply:
No, `--disable` does not need to be the same on all servers.
https://github.com/k3s-io/k3s/blob/bc5b42c27908ab430101eff0db0a0b22f870bd7a/pkg/daemons/config/types.go#L132C1-L150